e2e/testdata/app-inspect.golden

```diff
@@ -0,0 +1,54 @@
+{
+    "Metadata": {
+        "version": "1.1.0-beta1",
+        "name": "simple",
+        "description": "new fancy webapp with microservices",
+        "maintainers": [
+            {
+                "name": "John Developer",
+                "email": "john.dev@example.com"
+            },
+            {
+                "name": "Jane Developer",
+                "email": "jane.dev@example.com"
+            }
+        ]
+    },
+    "Services": [
+        {
+            "Name": "api",
+            "Image": "python:3.6",
+            "Replicas": 1,
+            "Mode": "",
```

Comment:
Should the empty values be omitted?

Reply:
I'm not really a fan of omitting empty values...

Reply:
I guess it's more concise 🤔

Reply:
Sounds  like ommiting empty is a _de facto_ convention, at least by docker